src/compiler/utils/string_with_sourcemap.ts

```diff
@@ -164,7 +167,17 @@ export class StringWithSourcemap {
 	}
 
 	static from_processed(string: string, map?: DecodedSourceMap): StringWithSourcemap {
-		if (map) return new StringWithSourcemap(string, map);
+		if (map) {
+			// ensure that count of source map mappings lines 
+			// is equal to count of generated code lines
+			// (some tools may produce less)
+			const missing_lines = string.split('\n').length - map.mappings.length;
```

Comment:
Makes sense to me but I have mixed thoughts on this...
- Is adding new line with source url really wrong? (I think this is majority of cases)
- Maybe svelte preprocessors should opt-out (if possible) or strip this line
- How should we report it? What if someone wants to suppress it?
- We could return it as warning (can be filtered later), but there are none in preprocess output...

Reply:
I would't want to print this message either because there is no good way currently to get this out to the user without possibility for him to surpress it. Also, this warning would be fired hundreds of times during one dev session.

Reply:
ok, im convinced

> broken tools should be fixed upstream

seems to be a de-facto-standard to append/attach annotation-comments, whose lines are not mapped in the sourcemap

thanks for the fix : )